src/main/java/frc/robot/subsystems/elevator/C2025ElevatorIO.java

```diff
@@ -29,7 +29,7 @@ public class C2025ElevatorIO implements ElevatorIO {
     private final DCMotorSim sim = new DCMotorSim(
             LinearSystemId.createDCMotorSystem(DCMotor.getKrakenX60(2),
                     0.001, GEAR_RATIO),
-            DCMotor.getKrakenX60(2).withReduction(1.0)
+            DCMotor.getKrakenX60(1).withReduction(1.0)
```

Comment:
Why was this changed? It shouldn't have been, the elevator still has 2 motors driving it.